static/src/stylesheets/module/external/_from-content-api.scss

```diff
@@ -116,6 +116,11 @@
 .from-content-api pre {
```

Comment:
have you chatted with the composer guys about this?  I think they would have an opinion - on first glance it does seem a bit broad, but if pre always wants to look like code then it's fine